events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,326 @@
+			value = flag:trapped_residence
+		}
+		
+		scope:target.court_owner.capital_province.barony = {
+			save_scope_as = court_location
+		}
+
+		stasis_trap_outcome_roll_setup_effect = yes
+	}
+
+	option = {
+		name = stasis_trap_outcome.1001.a
+		remove_short_term_gold = minor_gold_value
+		stasis_trap_outcome_roll_effect = yes
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.1001.b
+		custom_tooltip = do_not_execute_stasis_trap_tooltip
+
+		dont_excecute_scheme_effect = yes
+
+		ai_chance = {
+			base = 0
+		}
+	}
+}
+
+#####################################
+# SUCCESS OUTCOMES 2001 - 2999
+#####################################
+
+#######################
+# Seized Roadside
+#######################
+stasis_trap_outcome.2001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = stasis_trap_outcome.2001.desc
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = fear
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		if = {
+			limit = {
+				exists = scope:scheme_discovered
+			}
+			stasis_trap_success_effect = { DISCOVERED = yes }
+		} 
+		else {
+			stasis_trap_success_effect = { DISCOVERED = no }
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.2001.a
+		play_music_cue = "mx_cue_prison"
+		successful_stasis_trap_outcome_event_option_effect = yes
+	}
+}
+
+#################################
+# FAILURE OUTCOMES 4000 - 4999
+#################################
+stasis_trap_outcome.4001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.4001.opening
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.4001.owner_discovered
+			}
+			desc = stasis_trap_outcome.4001.owner_secret
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = rage
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		stasis_trap_failure_effect = yes
+	}
+
+	option = { # Abandon scheme
+		name = stasis_trap_outcome.4001.a
+		abandon_scheme_effect = yes
+
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = { # Try again!
+		name = stasis_trap_outcome.4001.b
+		restart_scheme_effect = yes
+
+		ai_chance = {
+			base = 10
+			modifier = {
+				add = 200
+				has_trait = stubborn
+			}
+		}
+	}
+}
+
+########################################
+# TARGET SUCCESS NOTIFICATIONS
+########################################
+
+#I have been stunned!
+stasis_trap_outcome.5001 = {
+	type = character_event
+	title = stasis_trap_outcome.5001.t
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					scope:scheme_discovered = yes
```

Comment:
I checked vanilla. Looks like it's an incorrect use.
Note: it's not the only wrong use in this file. Fix another as well.

Reply:
While I'm not going to resolve this conversation, I don't think this is a problem. Look at messages below.
![bild](https://user-images.githubusercontent.com/4825809/116904357-e29f3800-ac3d-11eb-9d61-cd4527b9b9e8.png)